releases.json

```diff
@@ -198,6 +198,14 @@
       "0.7.5-1"
     ]
   },
+  "fdk-aac": {
+    "dependency_names": [
+      "fdk-aac"
+    ],
+    "versions": [
+      "2.0.2",
```

Comment:
json is really annoying at not allowing the coma here.

Reply:
OOps,